src/utils/discordEphemeralResponse.ts

```diff
@@ -0,0 +1,11 @@
+import { InteractionResponseType } from "discord-interactions";
+import JSONResponse from "./JsonResponse";
+export const discordEphemeralResponse = (reply: string): JSONResponse => {
+  return new JSONResponse({
+    type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+    data: {
+      content: reply,
+      flags: 64,
```

Comment:
nit: please add a comment here specifying that flag 64 will send an ephemeral message so that it helps future devs